### ISSUE TITLE
Solutions layout-to-code

### ISSUE THREAD

**Issue by 007-hd** [007-hd/layout-to-code#2]
1. img src="..." alt"..." - image means there is a picture or image being added into the page - src is the link or path to the image - alt is the description of the image if it cannot be seen for accessibility 
2. h1 - the main title of the page - the most important information in terms of hierarchy 
3. figure - embedded annotated images or illustrations 
4. ul - this is an unordered list meaning that the order is not necessarily important of each item 
5. img src=" " alt" " - img means imbeds an image that is content specific - src is the path for the photo or link - alt is the description of the image for accessibility 
6. li - this is a singular item in the list 
7. figure - embedded annotated images or illustrations  
8. img src=" " alt" " - img means imbeds an image that is content specific - src is the path for the photo or link - alt is the description of the image for accessibility 
9.  img src=" " alt" " - img means imbeds an image that is content specific - src is the path for the photo or link - alt is the description of the image for accessibility 10. p - body copy or paragraph of the page 
10. p - this is a paragraph of of body copy on the page 

@abhaysharma

**Comment by abhaysharma** [comment]
Please submit the vocalizing of the code. Check the instructions for an example.
